docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -919,44 +962,46 @@ public interface HeroClient {
 
 }
 ----
-<1> Configure the base URI and the configuration key. 
+<1> Configure the base URI and the configuration key.
 The name must be in the format `<service-name>.<namespace>.svc`; otherwise, the certificate will not be trusted.
 Ensure that the `configKey` is also configured.
 
 . Configure the REST client to trust the CA certificate generated by OpenShift:
 +
-[source, properties]
+[source,properties]
 ----
 quarkus.rest-client.hero.tls-configuration-name=my-service-tls # <1>
 quarkus.tls.my-service-tls.trust-store.pem.certs=/deployments/tls/service-ca.crt # <2>
 ----
 <1> Configure the `hero` REST client with the TLS configuration named `my-service-tls`.
-<2> Set up the `my-service-tls` TLS configuration, specifically the truststore with the CA certificate. 
-Ensure the path matches the one used in the Kubernetes _Deployment_ configuration. 
+<2> Set up the `my-service-tls` TLS configuration, specifically the truststore with the CA certificate.
+Ensure the path matches the one used in the Kubernetes _Deployment_ configuration.
 The file is always named `service-ca.crt`.
 
 === Certificate renewal
 
 OpenShift automatically renews the certificates it generates.
 When the certificate is renewed, the secret is updated with the new certificate and private key.
 
-To ensure your application uses the new certificate, you can leverage the Quarkus TLS registry's periodic reloading feature.
+. To ensure your application uses the new certificate, you can leverage the Quarkus TLS registry's periodic reloading feature.
++
 By setting the `reload-period` property, the TLS registry will periodically check the key stores and trust stores for changes and reload them if needed:
-
-[source, properties]
++
+[source,properties]
 ----
 quarkus.tls.reload-period=24h
 ----
 
-You can also implement a custom mechanism to reload the certificates when the secret is updated.
+. Optionally, implement a custom mechanism to reload the certificates when the secret is updated.
 See <<reloading-certificates>> for more information.
```

Comment:
```suggestion
See xref:reloading-certificates[] for more information.
```

Reply:
Hello @jherrman  Jirka, We are not doing this manually. We keep the xrefs in this format and use a script for the automatic conversion.